src/Html.php

```diff
@@ -1161,6 +1161,10 @@ public static function includeHeader(
                 Html::requireJs('cable');
             }
 
+            //if (in_array('tablericons-definitions', $jslibs)) {
```

Comment:
Did you forget to un-comment ?

Reply:
This data probably shouldn't be imported like this at all. I suggest removing all of the on-demand loading stuff in this class and add an import inside the WebIconSelector.js file instead.

Reply:
This will also fix one of the Jest test failures.

Reply:
yes indeed